TESTS/mbed_hal/sleep/main.cpp

```diff
@@ -122,8 +122,29 @@ void deepsleep_lpticker_test()
 
         lp_ticker_set_interrupt(next_match_timestamp);
 
+        /* On some targets like STM family boards with LPTIM enabled there is a required delay (~100 us) before we are able to
+           reprogram LPTIM_COMPARE register back to back. This is handled by the low level lp ticker wrapper which uses LPTIM_CMPOK interrupt.
+           CMPOK fires when LPTIM_COMPARE register can be safely reprogrammed again. This means that on these platforms we have additional interrupt
+           (CMPOK) fired always ~100 us after programming lp ticker. Since this interrupt wake-ups the board from the sleep we need to go to sleep
```

Comment:
```suggestion
           (CMPOK) fired always ~100 us after programming lp ticker. Since this interrupt wakes up the board from the sleep we need to go to sleep
```

Reply:
Fixed.